docker/launch-hostagent.sh

```diff
@@ -37,6 +37,7 @@ EOF
    "name": "k8s-pod-network",
    "type": "opflex-agent-cni",
    "wait-for-network": true,
+   "wait-for-network-duration": $DURATION_WAIT_FOR_NETWORK,
```

Comment:
Put the following before L22:

  INT_DURATION_WAIT_FOR_NETWORK=210
  if [ -z != $DURATION_WAIT_FOR_NETWORK ]; then
    INT_DURATION_WAIT_FOR_NETWORK=$((DURATION_WAIT_FOR_NETWORK))
  fi

And then change above to:

   "wait-for-network-duration": $((INT_DURATION_WAIT_FOR_NETWORK)),